EPFL svg official icons

xentenza opened the pull request:
_[Updated and new icons](https://github.com/epfl-si/elements/pull/647/files)_ branch, reloaded.

- added EPFL official icons to the Styleguide.

xentenza committed:
EPFL svg official icons

added EPFL official icons to the Styleguide

williambelle commented on package.json:
```diff
@@ -169,6 +169,7 @@
     "jquery-mousewheel": "^3.1.13",
     "multiple-select": "^1.5.2",
     "object-fit-images": "^3.2.4",
+    "package.json": "^2.0.1",
```
C'est bizarre ce changement.

xentenza replied:
Plus rien ne fonctionnait en local, j'ai donc du faire un `npm install package.json`
Pourquoi ça a ajouté cette ligne, aucune idée. Toujours est-il que c'est supprimé

williambelle replied:
Oui, il y a toujours le problème. Les tests ne passent pas.

Il y a un conflit entre `yarn.lock` et `package-lock.json`.
Tu peux supprimer le fichier `package-lock.json` et annuler tous les changements du `yarn.lock`.

xentenza committed:
Update package.json

Correction commentaire https://github.com/epfl-si/elements/pull/672#pullrequestreview-1530500370

xentenza committed:
del package-lock.json, rev yarn.lock

-deleted package-lock.json
- revert changes yarn.lock
